src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -530,7 +558,22 @@ void cpu_convert(const void *srcPtr,
         IE_THROW() << "cpu_convert has null data pointer";
 
     if (srcPrc == dstPrc && srcPrc == interimPrc) {
-        cpu_memcpy(dstPtr, srcPtr, size * dstPrc.size());
+        const int L3_cache_size = mkldnn::utils::get_cache_size(3, true);
```

Comment:
Can we make sure that call get_cache_size() doesn't take significant amount of time? Otherwise if it is quite time consuming we have to move it outside of inference stage.

Reply:
Hi get_cache_size only does a case-switch inside,  I think the overhead is very small. 
```
    auto guess = [](int level) {
        switch (level) {
            case 1: return 32U * 1024;
            case 2: return 512U * 1024;
            case 3: return 1024U * 1024;
            default: return 0U;
        }
    };
```